js/app.js

```diff
@@ -1,20 +1,82 @@
 $(document).ready(function() {
-}
+	$('.start').on('click', function(e) {
+		e.preventDefault();
+		$('#intro-container').hide();
+		$('#question-container').show();
+	});
+	$('q-submit-button').on('click', function(e) {
+		e.preventDefault();
 
-// Answer 5 questions
+		quiz.checkAnswer();
+		//Hide question div
+		//Show answer div
+	});
+	$('nextButton').on('click', function(e) {
+		e.preventDefault();
 
-// Show each question one at a time
+		quiz.nextQuestion();
+		// Hide answer div
 
-// Tell user each question's number
+		// Show questions div
+	});
 
-// Show how many questions remain
+});
 
-// Prevent user from skipping questions
 
-// Gives user some way to answer each question
+var questions = [{
+	question: "What is the capital of Australia?",
+	choices: ["Sydney", "Canberra", "Melbourne", "Adelaide"],
+	correct: 0,
+	},
+	{
+	question: "By area, what is the smallest country on the planet?",
+	answers: ["Vatican City", "Monaco", "Nauru", "Madagascar"],
+	correct: 0,
+	},
+	{
+	question: "What is the name of the tallest uninterrupted waterfall in the world?",
+	choices: ["Victoria Falls", "Angel Falls", "Niagara Falls", "Alamere Falls"],
+	correct: 1,
+	},
+	{
+	question: "What is the correct spelling of this landlocked Asian nation?",
+	choices: ["Kyrgyzstan", "Kurjistan", "Kergystan", "Kyigyzstan"],
+	correct: 0,
+	},
+	{
+	question: "What country is home to the tallest mountain in the world, Mount Everest?",
+	choices: ["Pakistan", "India", "Bhutan", "Nepal"],
+	correct: 3,
+}];
 
-// Compare user's answer and the correct answer to determine score
+var quiz = {
+	questions: questions,
+	currentQuestion: 0,
+	score: 0,
+	nextQuestion: function() {
+		// Should first check to make sure we're not at the end
+		// Then increment this.currentQuestion
+		this.currentQuestion++;
+		// Call a method to write to the DOM
+	},
+	writeToDOM: function() {
+		// use questions and currentQuestion to write to DOM
+		var question = this.questions[this.currentQuestion];
+		$('answer-div').append(question.question);
+		question.choices.forEach(function(choice, i) {
+			$('q-radio')[i].append(choice);
+			// This may be the wrong way to work with radio in jQuery
+			// NOTE: http://stackoverflow.com/questions/596351/how-can-i-know-which-radio-button-is-selected-via-jquery
 
-// Allow user to start new game after questions are answered
-
-// Use objects to represent questions and answers
\ No newline at end of file
+		});
+	},
+	checkAnswer: function() {
+		// Will compare answer index to correct for current question
+		// If answer is correct increment this.score
+		var answer = $('q-radio').value();
+		// This may be the wrong way to work with radio in jQuery
+		if (answer === this.questions[this.currentQuestion].correct) {
+			//increment
```

Comment:
I just added the mustaches here for proper syntax to fix this error:

 
<img width="1920" alt="screen shot 2016-07-05 at 4 42 50 pm" src="https://cloud.githubusercontent.com/assets/8075821/16601570/00b8d320-42d3-11e6-865b-6c9ad4041f2f.png">

Obviously there's still no logic in there.